torch/csrc/jit/tracer.cpp

```diff
@@ -185,4 +185,15 @@ void setWarn(warn_fn_type fn) {
   warn_callback.store(fn);
 }
 
+void ensureUnique(const char * name, const at::Tensor& tensor) {
+  auto aliases = tensor.storage().use_count();
+  if (aliases > 1) {
+    std::stringstream ss;
+    ss << "There are " << aliases
+       << " live references to the tensor being modified when tracing in-place operator "
+       << name << " which ";
+    warn(ss.str().c_str());
```

Comment:
This will produce a very weird error message, because:
1. Ok, there are 5 references, but what does that imply for me as a user? Why is that bad? We should explain that in detail (say that we don't keep track of views/trace might be incorrect/mutation is not a good thing for tracing right now).
2. `warn` will complain that we can't trace the Python data flow, which is completely irrelevant, and might distract users from the real problem we should be describing here.